pyproject.toml

```diff
@@ -100,7 +100,7 @@ packages = ["elasticsearch_serverless"]
 
 [tool.pytest]
 junit_family = "legacy"
-addopts = "-vvv -p no:logging --cov-report=term-missing --cov=elasticsearch_serverless --cov-config=.pyproject.toml"
```

Comment:
The pytest-cov options are already specified in noxfile.py, and I like not having them when running pytest locally as coverage information is really verbose.